rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "1.76.0"
+components = ["rust-src"]
+profile = "default"
+targets = ["x86_64-pc-windows-msvc", "aarch64-linux-android", "aarch64-apple-darwin"]
```

Comment:
should windows and the apple target be in here by default? android is probably a target you want to cross compile to but windows and apple are probably your host system

Reply:
then we should also add linux